bigframes/functions/_function_session.py

```diff
@@ -775,6 +775,8 @@ def udf(
                 Name of the BigQuery connection. It is used to provide an
                 identity to the serverless instances running the user code. It
                 helps BigQuery manage and track the resources used by the udf.
+                When it defaults to None, the udf will be created without
+                referencing a specific connection in its BigQuery DDL.
```

Comment:
Nit: `DDL` isn't really relevant to Python users. Let's rephrase. Accounting for @GarrettWu's feedback, let's try this:

```suggestion
                any connection. A udf without a connection has no
                internet access and no access to other GCP services.
```

Reply:
Applied. Thanks.